src/library-authoring/data/api.ts

```diff
@@ -94,6 +91,28 @@ export interface CreateBlockDataResponse {
   tagsCount: number;
 }
 
+export interface UpdateBlockDataRequest {
```

Comment:
How is this "Block Data"? It seems like this should be called `UpdateLibraryDetailsRequest`

Reply:
Updated: https://github.com/openedx/frontend-app-course-authoring/pull/1138/commits/a4562d45894e01d7892359f57f3d46c3d6d73b0a